virtool/account/oas.py

```diff
@@ -262,6 +278,9 @@ class Config:
 
 
 class GetAPIKeysResponse(BaseModel):
+    id: str
+    name: str
```

Comment:
You sure that's all that is in this response?

Should be called `ListAPIKeysResponse`. If a reader misses the 's' at first glace, they may think this is a response model for getting a single API key.